pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -572,20 +587,32 @@ func (plugin *kubenetNetworkPlugin) checkCNIPluginInDir(dir string) bool {
 	return true
 }
 
-// Returns a list of pods running or ready to run on this node and each pod's IP address.
-// Assumes PodSpecs retrieved from the runtime include the name and ID of containers in
-// each pod.
-func (plugin *kubenetNetworkPlugin) getActivePods() ([]*hostport.ActivePod, error) {
+// getNonExitedPods returns a list of pods running or ready to run on this node
```

Comment:
It's just a very convoluted way to get a set of running infra containers, but I get the need to keep the change minimal for cherrypicking.